Sources/Testing/SourceAttribution/SourceContext.swift

```diff
@@ -41,3 +40,17 @@ extension SourceContext: Equatable, Hashable {}
 // MARK: - Codable
 
 extension SourceContext: Codable {}
+
+// MARK: - Deprecated
+
+extension SourceContext {
+  @available(*, deprecated, message: "Use init(backtrace:sourceLocation:) and pass both arguments explicitly instead.")
```

Comment:
One more needed with no args, right?

Reply:
The second deprecated overload below, `init(sourceLocation:)`, still includes a default value, so that covers the zero-argument usage scenario. Only two overloads needed!